futures-interact-cli/helpers.ts

```diff
@@ -0,0 +1,49 @@
+import { BigNumber, ethers } from "ethers";
+import { artifacts } from "hardhat";
+import { web3 } from "hardhat";
+const MockAggregator = artifacts.require("MockAggregatorV2V3");
+const ZERO_ADDRESS = "0x" + "0".repeat(40);
+
+const currentTime = async () => {
+  const { timestamp } = await web3.eth.getBlock("latest");
+  return timestamp;
+};
+
+export async function setupPriceAggregators(
+  exchangeRates: ethers.Contract,
+  deployerWallet: ethers.Wallet,
+  assetsBytes32: string[]
+) {
+  for (const assetBytes32 of assetsBytes32) {
+    const aggregator = await MockAggregator.new({
+      from: deployerWallet.address,
+    });
+
+    await exchangeRates
+      .connect(deployerWallet)
+      .addAggregator(assetBytes32, aggregator.address, {
+        from: deployerWallet.address,
+      });
+    /**
+     * await MockAggregator.new({ from: owner.address }); is causing the process to never end
```

Comment:
Sorry that comment was confusing.  I tried to make it more understandable. 
Basically when calling `MockAggregator.new` it looks like some intervals (or subscriptions) are being set up, meaning that the nodejs process doesn't end. 
`const MockAggregator = artifacts.require("MockAggregatorV2V3");` `artifacts.require comes from `@nomiclabs/hardhat-truffle5` so I think it might be a truffle problem.  And as my comment states, I tried calling `clearSubscriptions` but that threw an error